packages/itmat-ui-react/src/setupProxy.js

```diff
@@ -44,4 +44,13 @@ module.exports = function (app) {
             ws: true
         })
     );
+    app.use(
+        '/public',
+        createProxyMiddleware({
+            target: API_SERVER,
+            changeOrigin: true,
+            autoRewrite: true,
+            ws: true
+        })
+    );
```

Comment:
What is prompting this addition ? Are there issues with the way CRA handles proxying of assets ?